activemodel/lib/active_model/type/boolean.rb

```diff
@@ -35,11 +26,9 @@ def serialize(value) # :nodoc:
 
       private
         def cast_value(value)
-          if value == ""
-            nil
-          else
-            !FALSE_VALUES.include?(value)
-          end
+          return nil if value == ""
+
+          !FALSE_VALUES.include?(value&.to_s&.downcase)
```

Comment:
This will allocate 1-2 additional strings every time a value is cast (depending on whether the value is already a string or not).  I believe the reason `FALSE_VALUES` was written as it was, was to avoid such allocations.  If supporting `"False"` and `"Off"` turns out to be reasonable, it might be best to add them to `FALSE_VALUES` instead.

Reply:
I think that sounds totally reasonable. I’ll update this PR after lunch!